kuma/static/js/payments-handler.js

```diff
@@ -458,7 +467,23 @@
         }
     }
 
+    /**
+     * Builds correct URL param and directs to GitHub for authentication.
+     * @param {jQuery.Event} event Event object.
+     */
+    function redirectUserToLogin(event) {
+        event.preventDefault();
+        var gitHubLink = $(this).attr('href');
+        var gitHubNext = $(this).data('next');
+        var getFormFields = form.serialize();
+        gitHubLink += '&next='+ gitHubNext + '?' + encodeURIComponent(getFormFields);
```

Comment:
Nit: space after '&next='

```suggestion
        gitHubLink += '&next=' + gitHubNext + '?' + encodeURIComponent(getFormFields);
```